spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
```

Comment:
I just don't know how to change the mailer's behavior in spec. The difficulty is described above. Should I modify my code to support this kind of test? @lowjoel